README.md

```diff
@@ -6,40 +6,76 @@ This agent monitors various aspects of UMA. The UMA suite currently contains
 the following handlers:
 
 - deployer-watch
+- admin-events
 - monitor-mint-calls
+- optimistic-oracle
 
 ## Supported Chains
 
 - Ethereum
 
-
 ## Alerts
 
+<!-- -->
 - AE-UMA-DEPLOYER-TX
   - Fired when the UMA Deployer contract is part of a transaction
   - Severity is always set to "low"
   - Type is always set to "unknown"
   - Metadata field contains to and from addresses
 
+<!-- -->
 - AE-UMA-DEPLOYER-WHITELIST
   - Fired when the UMA Deployer interacts with a non-whitelisted address
   - Severity is always set to "high"
   - Type is always set to "suspicious"
   - Metadata field contains to and from addresses
 
+<!-- -->
 - AE-UMA-ADMIN-EVENT
   - Fired on any event in admin-events.json
   - Severity is set to value in admin-events.json
   - Type is set to value in admin-events.json
   - Metadata field contains contract name, contract address and event name
 
+<!-- -->
 - AE-UMA-UNAUTHORIZED-MINT
   - Fired when the UMA VotingToken.mint() method is called by any address other than the UMA Voting contract
   - Severity is always "critical"
   - Type is always set to "exploit"
   - Metadata field contains VotingToken contract address, caller address, and transaction hash
   - Must have traces enabled for this to operate correctly
 
+<!-- -->
+- AE-UMA-OO-REQUESTPRICE
+  - Fired when the RequestPrice event is emitted by the Optimistic Oracle contract
+  - Severity is always set to "low"
+  - Type is always set to "info"
+  - Metadata field contains requester, identifier, and price (obtained from external price feed)
+
+<!-- -->
+- AE-UMA-OO-PROPOSEPRICE
+  - Fired when the ProposePrice event is emitted by the Optimistic Oracle contract
+  - Severity is set to "low" for a price that is within the threshold of the price feed data,
+    and set to "high" for a price that exceeds the threshold
```

Comment:
Not required, but a formatting change that might help visually indicate how many severity levels there are for this handler.
```suggestion
  - Severity is set to:
    - "low" for a price that is within the threshold of the price feed data
    - "high" for a price that exceeds the threshold
```

Reply:
Changed